modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,18 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for creating service account resources
+[%collapsible]
+====
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+[IMPORTANT]
+====
+These permissions are required when the user does not supply a service account for compute and control plane nodes in the install config. The permissions should be granted in the host project.
```

Comment:
```suggestion
The following permissions are required if you do not supply a service account for compute and control plane nodes in the `install-config.yaml` file. The permissions must be granted in the host project.
```

I would recommend changing this to "the following permissions" and move the note to right before line 28, so customer reads the note before reading the permissions.